arm/Microsoft.Network/virtualNetworks/virtualNetworkPeerings/deploy.bicep

```diff
@@ -31,7 +31,7 @@ module pid_cuaId '.bicep/nested_cuaId.bicep' = if (!empty(cuaId)) {
 }
 
 resource virtualNetworkPeering 'Microsoft.Network/virtualNetworks/virtualNetworkPeerings@2021-02-01' = {
-  name: '${localVnetName}/${peeringName}'
+  name: '${localVnetName}/${name}'
```

Comment:
Please add `@description('...')` for outputs and regenerate the docs like described in the other readme comment

Reply:
done